qos-service-api/src/main/java/com/palantir/atlasdb/qos/QosServiceResource.java

```diff
@@ -17,8 +17,19 @@
 package com.palantir.atlasdb.qos;
 
 public class QosServiceResource implements QosService {
+
+    private QosServiceRuntimeConfig config;
+
+    public QosServiceResource(QosServiceRuntimeConfig config) {
+        this.config = config;
+    }
+
     @Override
-    public int getLimit(String client) {
-        return Integer.MAX_VALUE;
+    public long getLimit(String client) {
```

Comment:
Any reason for int->long? just generally giving ourselves more room?

Reply:
Exactly, just to give us more room.